### ISSUE TITLE
Swift Package Manager

### ISSUE THREAD

**Issue by TheLDB** [gianmode1803/GMTextField#1]
Could you consider making a swift package for this to support Swift Package Manager?

**Comment by gianmode1803** [comment]
Yes of course! I will do it in the next days/weeks hehe sorry ;)
